src/test/java/com/ionosenterprise/rest/test/IPBlockTest.java

```diff
@@ -104,8 +110,62 @@ public void t5_createIpBlockFail() throws IOException, IllegalAccessException, I
         }
     }
 
+    @Test
+    public void t5_testIpBlockConsumerDetails() throws InvocationTargetException, NoSuchMethodException,
+            IllegalAccessException, RestClientException, IOException, InterruptedException {
+        DataCenter newDatacenter = ionosEnterpriseApi.getDataCenter().createDataCenter(
+                DataCenterResource.getDataCenter());
+        assertNotNull(newDatacenter);
+        waitTillProvisioned(newDatacenter.getRequestId());
+        dataCenterId = newDatacenter.getId();
+
+        Server server = new Server();
+        server.getProperties().setName("SDK TEST SERVER");
+        server.getProperties().setRam(1024);
+        server.getProperties().setCores(1);
+        Server newServer = ionosEnterpriseApi.getServer().createServer(dataCenterId, server);

```

Comment:
Please use ServerResource.getServer() or define a custom method in ServerResource so that we can get read of the Server creation boilerplate code.

Please do the same for the rest of resources e.g. Lan, Nic etc

Reply:
Done.